examples/config.hocon.kinesis.minimal

```diff
@@ -0,0 +1,25 @@
+collector {
+  interface = "0.0.0.0"
+  port = 80
+
+  streams {
+    good = "{{good}}"
+    bad = "{{bad}}"
+
+    sink {
+      enabled = kinesis
+      region = "{{region}}"
+      threadPoolSize = 10
+
+      aws {
+        accessKey = iam
+        secretKey = iam
+      }
+
+      backoffPolicy {
+        minBackoff = 3000
+        maxBackoff = 600000
```

Comment:
Same, I think we can be opinionated about these values and set sensible defaults.

Reply:
`3000` and `600000` are good defaults.